src/test/java/net/masterthought/cucumber/json/EmbeddingWithNameTest.java

```diff
@@ -0,0 +1,126 @@
+package net.masterthought.cucumber.json;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assumptions.assumeThat;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+/**
+ * @author Damian Szczepanik (damianszczepanik@github)
+ * @author <a href="https://github.com/ghostcity">Stefan Gasterstädt</a>
```

Comment:
I wonder if this multiple @author is a valid javadoc format :)

Reply:
To the best of my knowledge it is. I even checked before doing so ;-)
> A documentation comment can contain multiple @author tags.
- https://docs.oracle.com/en/java/javase/13/docs/specs/javadoc/doc-comment-spec.html#author